platform/core.network/src/org/netbeans/core/network/proxy/pac/impl/NbPacScriptEvaluator.java

```diff
@@ -384,29 +400,14 @@ private PacJsEntryFunction testScriptEngine(ScriptEngine eng, boolean doDeepTest
     }
 
     private boolean isJsFunctionAvailable(ScriptEngine eng, String functionName, boolean doDeepTest) {
-        // We want to test if the function is there, but without actually 
-        // invoking it.        
```

Comment:
Consider keeping this comment as it explains the purpose of the method.

Reply:
Returned back in afc27a44b0a7